packages/file-storage-node/src/lib.ts

```diff
@@ -163,7 +163,7 @@ export async function largeFileDownloadNode(
   const encryptionArtifacts: any = await AzureBlockDownload.download(
     artifactsUrl,
     authConfig
-  ).then((result: any) => JSON.parse(result.toString('utf-8')));
+  ).then((result: any) => JSON.parse(bytesToUtf8(result)));
```

Comment:
Ah I added this fix before I saw the PR. Updated my change to reflect the fact default encoding of fs.write is utf8